docs/conceptual/node/extrinsics.md

```diff
@@ -2,12 +2,91 @@
+runtime. The extrinsics root is a cryptographic digest of this series. This serves two purposes. 
+First, it prevents any alterations to the series of extrinsics after the header has been built and 
+distributed. Second, it provides a means of allowing light clients to succinctly verify that any 
+given extrinsic did indeed exist in a block given only knowledge of the header.
+
+- [Block Reference](https://crates.parity.io/sp_runtime/traits/trait.Block.html)
+
+## Inherents
+
+Inherents are pieces of information that are not signed and only inserted into a block by the block 
+author. They are not gossiped on the network or stored in the transaction queue. There is nothing 
+technically preventing a Substrate chain that gossips inherents, but there would be no fee-based 
+spam prevention mechanism.
+
+Inherents represent data that, in an opinionated fashion, describes one of a number of valid pieces 
+of information. They are assumed to be "true" simply because a sufficiently large number of 
+validators have agreed on them being reasonable.
+
+For example, the author of the block may insert a timestamp inherent into the block. There is no 
+way to prove that a timestamp is true the way the desire to send funds is proved with a signature. 
+Rather, validators accept or reject the block based on how reasonable the other validators find the 
+timestamp, which may mean it is within some acceptable range of their own system clocks.
+
+- [Inherents Reference](https://crates.parity.io/sp_inherents/index.html)
+
+## Signed Transactions
+
+Signed transactions contain a signature of the account that issued the transaction and stands to 
+pay a fee to have the transaction included on chain. Because the value of including signed 
+transactions on-chain can be recognized prior to execution, they can be gossiped on the network 
+between nodes with a low risk of spam.
+
+Signed transactions fit the concept of a transaction in Ethereum or Bitcoin.
+
+## Unsigned Transactions
+
+Some cases call for unsigned transactions. Use unsigned transactions with care, as their validation 
+logic can be difficult.
+
+Since the transaction is not signed, there is nobody to pay a fee. Because of this, the transaction 
+queue lacks economic logic to prevent spam. Unsigned transactions also lack a nonce, making replay 
+protection difficult. A few transactions warrant using the unsigned variant, but they will require 
+some form of spam prevention.
+
+An example of unsigned transactions in Substrate is the 
+[I'm Online](https://github.com/paritytech/substrate/blob/master/frame/im-online/src/lib.rs) 
+heartbeat transaction sent by authorities. The transaction includes a signature from a Session key, 
+which does not control funds and therefore cannot pay a fee. The transaction pool controls spam 
+based on an implementation of `validate_unsigned` that checks if a heartbeat has already been 
```

Comment:
Since `validate_unsigned` is already deprecated and not recommended I'd add a link to `Signed Extension` here and then add a paragraph that describes the usage for validating unsigned piece of data.